metricbeat-yml-script.py

```diff
@@ -40,44 +41,37 @@ def _is_open():
 
 
 def _add_modules():
-    if _custom_modules():
-        logger.debug("Using custom modules")
-        return
     try:
         modules = [m.strip() for m in os.environ["LOGZIO_MODULES"].split(",")]
     except KeyError:
         logger.error("Required at least one module")
         raise
-
     _enable_modules(modules)
 
 
-def _custom_modules():
-    return len(glob.glob("{}/*.yml".format(MODULES_DIR)))
-
-
 def _enable_modules(modules):
     yaml = YAML()
-    supported_modules = dict((name, setup) for name, setup in setups)
+    yaml.preserve_quotes = True
```

Comment:
Maybe this should be `True` only in the modules that need that

Reply:
Why? I only want change enabled from false to true, if the values had quotes I want to keep it that way